news/2022-08-30-changes-to-osu-sr-and-pp.md

```diff
@@ -0,0 +1,98 @@
+
+The system is now based on comparing the 300 hitwindow size to the time between previous and current object, and whether the player has enough breathing room after the current note to be able to comfortably double tap in the first place. Being able to access the next note has only been recently possible through a refactor effort, hence why this wasn't done in the first place.
+
+The change may be easier digested an interactive graphical form, so feel free to play around with [a Desmos implementation of the system](https://www.desmos.com/calculator/zl1hfqd9hm).
+
+### Flashlight changes
+
+The incredible illuminator [MBmasher](https://osu.ppy.sh/users/4498616) continues his focused efforts on improving Flashlight difficulty calculation, yielding a series of changes to the Flashlight skill that was introduced in the last deployment.
+
+#### Hit object opacity
+
+The opacity of an object plays a large role in Flashlight difficulty, since it can be the line between reading and memorising the object. With [this proposal](https://github.com/ppy/osu/pull/15665), the opacity of a hit object is a factor in Flashlight difficulty.
+
+An object's opacity is affected by it's Approach Rate and whether the Hidden mod is applied. A low Approach Rate coupled with Hidden means that objects can be invisible for a significant amount of time, and a very high Approach Rate (especially ones found with DT) means that an object is visible for barely any time at all. Both of these scenarios now apply a bonus to Flashlight difficulty!
+
+**Note that adding Hidden to Flashlight will now affect Star Rating.**
+
+#### Repeated Angles
+
+Maps with Flashlight applied become significantly easier if the successive object placement is predictable. One of the major ways this can manifest is through the use of repeated angles, which [this change](https://github.com/ppy/osu/pull/19716) aims to nerf.
+
+Throughout the development process for the Flashlight changes, a number of grid-style maps such as [DJPop's Night of Knights [SOLO]](https://osu.ppy.sh/beatmapsets/15920#osu/58063) became increasingly concerning as their values rose. By nerfing repeat angles, grid style maps such as this become nerfed to a more suitable value.
+
+#### Slider bonus
+
+Fast and long sliders introduce additional difficulty to Flashlight play, so a slider bonus is [now applied](https://github.com/ppy/osu/pull/15666)! The length of a slider wasn't considered at all up till now.
+
+The bonus only kicks in once a slider is fast enough since (slow sliders are easily readable), and the bonus also has a distance factor to account for longer sliders.
```

Comment:
```suggestion
The bonus only kicks in once a slider is fast enough since slow sliders are easily readable, and the bonus also has a distance factor to account for longer sliders.
```